src/Common/src/TypeSystem/IL/Stubs/CalliIntrinsic.cs

```diff
@@ -45,7 +45,8 @@ public static MethodIL EmitIL(MethodDesc target)
 
             var signature = new MethodSignature(template.Flags, 0, returnType, parameters);
 
-            codeStream.Emit(ILOpcode.calli, emitter.NewToken(signature));
+            //codeStream.Emit(ILOpcode.calli, emitter.NewToken(signature));
```

Comment:
We may want to skip this for `RawCalliHelper` - similar to what the transform in NUTC does.